### ISSUE TITLE
Eternal Warcry / Universal Tracker mismatch

### ISSUE THREAD

**Issue by joeymavity** [extesy/DeckTracker#59]
So I'm not sure who should 'fix' this but:

The export deck counts my normal "Madness" card as 1 card, and then re-lists "Madness" again 1 time for my foil copy. When that gets imported into the EternalWarcry website, it only counts one of the madnesses, both within my collection and then also when calculating how much Shiftstone I'm short.

My sense is that DeckTracker should report me as having 2 madness cards, regardless of type, but I can think of good arguments against that, too. Ultimately, it would be nice if DeckTracker & EternalWarcry handled this better, so I had a better sense of which decks I was close to. 

(nb: Importing works fine: DeckTracker will put both of my madness cards into an imported deck, the way it should - kudos on that end.)

Thanks.

**Comment by joeymavity** [comment]
While doing some troubleshooting / testing, I noticed it's a little worse than I described above. EternalWarcry is taking the FIRST card listed as the total number of cards. So for instance, I have 1 foil "ornamental daggers" and 3 regular "ornamental daggers". The 1 foil is listed first ([line 108](https://pastebin.com/CvjngJUG)), and so EW thinks I only have 1, even though the 3 regular ones show up on line 150. Thanks for taking a look at this one - the quick workaround is probably to just sum the various types, until EW starts tracking the two separately.
